example/mbed-os-5/main.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdio.h>
+
+#include "mbed.h"
+
+#define TRACE_GROUP                 "main"
+#define DEFAULT_SERIAL_BAUD_RATE    (115200)
+
+static RawSerial pc(USBTX, USBRX, DEFAULT_SERIAL_BAUD_RATE);
```

Comment:
I don't think RawSerial should be used. Better to just define the baud rate in the app config file.